internal/infrastructure/storage/db/test/utxo_repository_test.go

```diff
@@ -69,10 +70,10 @@ func testAddUtxos(t *testing.T, repo domain.UtxoRepository) {
 
 func testGetUtxos(t *testing.T, repo domain.UtxoRepository) {
 	t.Run("get_utxos", func(t *testing.T) {
-		utxos := repo.GetAllUtxos(ctx)
+		utxos, err := repo.GetAllUtxos(ctx)
```

Comment:
Let's make sure no error is raised.
```suggestion
		utxos, err := repo.GetAllUtxos(ctx)
		require.NoError(err)
```

Reply:
Done